src/common/http.cc

```diff
@@ -455,10 +465,18 @@ namespace Pistache::Http
                     if (!cursor.advance(1))
                         return Incomplete;
 
-                char* end;
-                const char* raw = chunkSize.rawText();
-                auto sz         = std::strtol(raw, &end, 16);
-                if (*end != '\r')
+                const char* beg = chunkSize.rawText();
+                const char* beg_trim = beg;
+                while (*beg_trim == '+' || *beg_trim == ' ')
+                {
+                    if (*(beg_trim + 1))
+                        ++beg_trim; 
+                }
+                std::string_view view = beg_trim;
+                const char* end = view.data() + view.size();
```

Comment:
I'm generally a fan of string_view, but here I personally find it unnecessary. I'd suggest using `strlen()` explicitly, like:

```suggestion
                const char* end = beg_trim + std::strlen(beg_trim);
```